### ISSUE TITLE
Error message when uninstalling

### ISSUE THREAD

**Issue by mikeSimonson** [pyrech/composer-changelogs#14]
Hi,

Here under the output from composer when I tried to uninstall composer-changelogs (to check that the command worked").
It worked in way because the package got removed but I also got a fatal error because a file that exists wasn't found.

I couldn't find were the error came from.
```
composer global remove "pyrech/composer-changelogs"
Changed current directory to /home/mike/.composer
Loading composer repositories with package information
Updating dependencies (including require-dev)
  - Removing pyrech/composer-changelogs (v1.1.1)
Writing lock file
Generating autoload files
PHP Fatal error:  Class 'Pyrech\ComposerChangelogs\Version' not found in /home/mike/.composer/vendor/pyrech/composer-changelogs/src/OperationHandler/UninstallHandler.php on line 54

Fatal error: Class 'Pyrech\ComposerChangelogs\Version' not found in /home/mike/.composer/vendor/pyrech/composer-changelogs/src/OperationHandler/UninstallHandler.php on line 54
```

**Comment by pyrech** [comment]
Hi, thanks for your issue. Yes, this is a due to the fact that the plugin is registered by Composer at the beginning of the command (so some classes are already loaded by PHP). But after the package was removed, Composer doesn't take it into account and try to run it normally so you get some weird issues.

This problem can also relate to #10 when some classes are removed in a new version of the plugin.

May be your specific issue could be solved directly in Composer (something like not running a plugin that was just removed). Anyway, your issue doesn't cause any problem to your installation (because the plugin run at the very end) so your lock file and yours vendors are safes ;-)

**Comment by pyrech** [comment]
Finally found a way to fix this kind of issues on my side (because Composer itself is not going to fix this bug soon). Once #22 will be released (probably in 1.3.0 in a few days), all the classes will be autoloaded when the plugin is enabled. Not the cleanest solution but should just work whenever the plugin is updated, removed or whatever.
